aligner/u_of_michigan_aligner/u_of_michigan_aligner.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
You should add a new JSON input file with the gs:// paths. Brian O'Conner requested a test parameter file with https paths.